regparser/commands/current_version.py

```diff
@@ -14,7 +14,7 @@
 logger = logging.getLogger(__name__)
 
 
-def process_if_needed(cfr_title, cfr_part, year):
+def process_if_needed(cfr_title, cfr_part, year, publication_date):
```

Comment:
The overlap/redundancy between year and publication_date is unsettling.
Wondering if it would not be better to change the signature drastically and have it be:
```
def process_if_needed(volume):
```